azure-pipelines.yml

```
@@ -1,6 +1,6 @@
 jobs:
-- job: lint
-  displayName: 'Linting with flake8'
+- job: lint_and_test
+  displayName: 'Lint with flake8 & test with pytest'
```

I don't think it's useful to put such details in the name. It gets cut off anyway when viewing it on the Azure build page.
```suggestion
  displayName: 'Lint & Test'
```